djsf.json

```diff
@@ -562,6 +562,12 @@
 					"description": "The sequence number of the station on the route.",
 					"id": "#sequenceNumber",
 					"required":true
+				},
+				"externalIdentifier": {
+					"type": "string",
+					"description": "A unique identifier for the station, formed by concatenating the station's idCode with the train's arrival track number. For example, APPZ_1.",

```

Comment:
Sorry, but the description is wrong.
This can be any string and can depend on the project it is being used. At CJ it will be one of the numbers of the `REC_ORT` table:
<img width="1331" alt="image" src="https://github.com/CN-Consult/djsf/assets/957556/55dad7a9-2390-4d9c-8512-12fd2cb081bc">

So for the documentation you should write that this could be any external identifier, for example the REC_ORT number of a VDV452 schedule Import.